authorizations/authorization-common/src/main/java/org/apache/gravitino/authorization/jdbc/JdbcAuthorizationPlugin.java

```diff
@@ -119,19 +119,18 @@ public Boolean onRoleCreated(Role role) throws AuthorizationPluginException {
     List<String> sqls = getCreateRoleSQL(role.name());
     boolean createdNewly = false;
     for (String sql : sqls) {
-      createdNewly = executeUpdateSQL(sql, "already exists");
+      createdNewly |= executeUpdateSQL(sql, "already exists");
```

Comment:
I don't think thi is correct, as using or means that if only one it true then the result is true. I think you want to initially set it to true and use "and" not "or". Also probably best not to use bitwise operators.

Reply:
fixed